app/views/users/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head></head>
+  <body>
+    <div class='container'>
+      <div class='card'>
+        <img class='userpic' src= <%= @user.photo %> >
+        <div class='leftcard'>
+          <h2 class='username'><%= @user.name %></h2>
+          <p class='count'>Number of posts: <%= @user.posts_counter >= 0 ? @user.posts_counter : '0' %></p><br>
+        </div>
+      </div>
+      <div class='biodiv'>
+        <p>Bio</p>
+        <p><%= @user.bio %></p>
+      </div>
+      <% @posts.each do |post| %>
+        <div class='postcard'>
+          <p><%= post.title %></p>
+          <p><%= post.text.truncate(50, separator:' ') %></p>
+          <p class='count'>
+            Comments: <%= post.comments_counter != nil ? post.comments_counter : '0' %>
+            Likes: <%= post.likes_counter != nil ? post.likes_counter : '0' %>
+          </p>
+        </div>
+      <% end %>
+      <a class='button' href=<%= user_posts_url(@user.id)%>>See All Posts</a>
+    </div>
```

Comment:
- [ ] As stated by the previous reviewer, at the moment when a user is clicked, it doesn't bring up the show page because of the url which is something like this `http://127.0.0.1:3000/users.1` except it is manually changed to `http://127.0.0.1:3000/users/1`.
Please ensure that it navigates to the show page when it is clicked